crates/bevy_pbr/src/bundle.rs

```diff
@@ -1,8 +1,9 @@
+#![expect(deprecated)]
```

Comment:
I tried to move this to the specific places it was needed in this file but then i stumbled upon https://github.com/bevyengine/bevy/issues/15531, like you probably did q: